powerauth-java-crypto/src/main/java/io/getlime/security/powerauth/crypto/lib/enums/PowerAuthSignatureFormat.java

```diff
@@ -37,7 +40,7 @@
      *     <li>For all versions of offline signatures.</li>
      * </ul>
      */
-    DECIMAL,
+    DECIMAL("DECIMAL"),
```

Comment:
Isn't the value a bit duplicate when you can get the same value using `toString()`?

Reply:
OK